### PR TITLE
Fix user specify in matview_ao test

### PR DESCRIPTION
**Pull request by bzhaoopenstack**
This patch will create a test role to exec the test.

Related-Issues: https://github.com/greenplum-db/gpdb/issues/10198

### CODE REVIEW

File: src/test/regress/expected/matview_ao.out [hunk 1]
```diff
@@ -1,3 +1,5 @@
+CREATE ROLE matview_ao_role;
+SET ROLE matview_ao_role;
```
Comment (paul-guo-) [inline src/test/regress/expected/matview_ao.out:2]: Wait. Better drop the role in the test end. Sometimes people want to rerun a test to debug.

**Comment by d** [comment]
LGTM.

I'm a bit upset about this test though, it's not the fault of this patch: If we take half a step back, one cannot help but ask: why _did_ we have a test that depends on the current user? What is the test trying to test? Is `\dm` a good way to achieve that intent?

File: src/test/regress/expected/matview_ao.out (hunk 1 above)
Comment (bzhaoopenstack) [reply src/test/regress/expected/matview_ao.out:2]: Yeah, thanks paul. I know that, we'd better to remove the role, but I'm thinking about not to change too much, as I found that the prevous behaviour about this sql test is NOT remove anything. And in the new change, if we change the test role, we need to drop every thing during testing which the role created. So for keeping the original behavior, that's why I did like this.

Comment (paul-guo-) [reply src/test/regress/expected/matview_ao.out:2]: OK. Pushed. Thanks for fixing.

**Pull request merged**